src/lib/nodes/btc/BtcNode.ts

```diff
@@ -29,4 +41,18 @@ export class BtcNode extends Node {
       ping: Date.now() - time
     }
   }
+
+  private async fetchNodeVersion(): Promise<void> {
+    try {
+      const { data } = await this.client.post<FetchBtcNodeInfoResult>('/bitcoind', {"jsonrpc":"1.0","id":"adm","method":"getnetworkinfo","params":[]})
+      const v =  data?.result?.version
```

Comment:
1. Use more descriptive variable names
2. You can omit optional chaining operator since `type FetchBtcNodeInfoResult` mark it as required

```suggestion
      const version =  data.result.version
```